internal/action/stash.go

```diff
@@ -212,3 +212,16 @@ func (b *Builder) switchTab(tab int) {
 	b.HID.Click(game.LeftButton, x, y)
 	helper.Sleep(500)
 }
+
+func (b *Builder) SwitchTab(tab int) *Chain {
```

Comment:
```suggestion
func (b *Builder) SwitchStashTab(tab int) *Chain {
```

Builder struct is shared across multiple parts of the application, better to be specific.